appcheck/firebase-appcheck/src/main/java/com/google/firebase/appcheck/internal/NetworkClient.java

```diff
@@ -131,17 +124,14 @@ public AppCheckTokenResponse exchangeAttestationForAppCheckToken(
       urlConnection.setDoOutput(true);
       urlConnection.setFixedLengthStreamingMode(requestBytes.length);
       urlConnection.setRequestProperty(CONTENT_TYPE, APPLICATION_JSON);
-      String userAgent = getUserAgent();
-      if (userAgent != null) {
-        urlConnection.setRequestProperty(X_FIREBASE_CLIENT, userAgent);
-      }
+      try {
+        String heartBeatHeader = getHeartBeat();
+        if (heartBeatHeader != null) {
+          urlConnection.setRequestProperty(X_FIREBASE_CLIENT, heartBeatHeader);
+        }
 
-      // getHeartbeatCode should not be called multiple times, as subsequent calls will return
-      // HeartBeat.NONE until the heartbeat is reset.
-      HeartBeat heartBeat = getHeartbeatCode();
-      if (heartBeat != HeartBeat.NONE) {
-        urlConnection.setRequestProperty(
-            X_FIREBASE_CLIENT_LOG_TYPE, Integer.toString(heartBeat.getCode()));
+      } catch (Exception e) {
+        Log.w(TAG, "Unable to get heartbeats!");
```

Comment:
This doesn't need to be wrapped in a try-catch, right? `getHeartBeat()` doesn't throw an exception